lib/rules/index.js

```diff
@@ -1,5 +1,214 @@
-'use strict';
+// STOP: This file is autogenerated. Do not modify by hand.
```

Comment:
This file now has to be auto-generated by a new script `yarn update:indices` since `requireIndex` doesn't work with ESM imports.